tests/functional/metadata/mongodb/listLifecycleObject/current.spec.js

```diff
@@ -306,18 +313,6 @@ describe('MongoClientInterface::metadata.listLifecycleObject::current', () => {
         });
     });
 
-    it('Should return an Internal error when bucket non existent (current)', done => {
```

Comment:
I removed this test because testing for non-existent bucket is not relevant at the Arsenal stage since the check/test is already done in CloudServer.